### PR TITLE
fix: refactor events stanza and adding allow-deny filter

### PR DESCRIPTION
**Pull request by msalman899**

### CODE REVIEW

File: charts/boundary-chart/templates/configmap.yaml [hunk 1]
```diff
@@ -52,16 +52,34 @@ data:
...
-        format = {{ $stderr_sink.format | default "hclog-json" | quote }}
-        {{- if (and (hasKey $stderr_sink "audit_filter_overrides")) }}
+      {{- range $sink := .Values.events.sink }}
+      sink {
+        type = {{ $sink.type | default "stderr" | quote }}
+        name = {{ $sink.name | quote }}
+        description = {{ $sink.description | quote  }}
+        event_types = {{ $sink.event_types }}
+        {{- if (and (hasKey $sink "evenevent_source_url")) }}
+        evenevent_source_url = {{ $sink.evenevent_source_url }}
```
Comment (janikgar) [inline charts/boundary-chart/templates/configmap.yaml]: This should be `event_source_url` throughout

File: charts/boundary-chart/templates/configmap.yaml [hunk 2]
```diff
@@ -155,16 +174,34 @@ data:
...
-        event_types = {{ $stderr_sink.event_types }}
-        format = {{ $stderr_sink.format | default "hclog-json" | quote }}
-        {{- if (and (hasKey $stderr_sink "audit_filter_overrides")) }}
+      {{- range $sink := .Values.events.sink }}
+      sink {
+        type = {{ $sink.type | default "stderr" | quote }}
+        name = {{ $sink.name | quote }}
+        description = {{ $sink.description | quote  }}
+        event_types = {{ $sink.event_types }}
+        {{- if (and (hasKey $sink "evenevent_source_url")) }}
```
Comment (janikgar) [inline charts/boundary-chart/templates/configmap.yaml]: `event_source_url`

**Review by janikgar — CHANGES_REQUESTED** [review]
Good change; I like making the sinks configurable with the `stderr` default.

If you can adjust the misspelling on `evenevent` where it shows up, it should be good so it will be compliant HCL. Thanks!

File: charts/boundary-chart/templates/configmap.yaml (hunk 1 above)
Comment (msalman899) [reply charts/boundary-chart/templates/configmap.yaml]: My bad, Fixed it. Thanks for checking.

File: charts/boundary-chart/templates/configmap.yaml (hunk 2 above)
Comment (msalman899) [reply charts/boundary-chart/templates/configmap.yaml]: Fixed.

**Review by janikgar — APPROVED** [review]

**Pull request merged**